bigtable/internal/testproxy/proxy.go

```diff
@@ -556,7 +557,60 @@ func (s *goTestProxyServer) ReadRow(ctx context.Context, req *pb.ReadRowRequest)
 // ReadRows responds to the ReadRows RPC. This method gets all of the column
 // data for a set of rows, a range of rows, or the entire table.
 func (s *goTestProxyServer) ReadRows(ctx context.Context, req *pb.ReadRowsRequest) (*pb.RowsResult, error) {
-	return nil, stat.Error(codes.Unimplemented, "ReadRows not implemented")
+	s.clientsLock.Lock()
+	btc, exists := s.clientIDs[req.ClientId]
+	s.clientsLock.Unlock()
+
+	if !exists {
+		return nil, stat.Error(codes.InvalidArgument,
+			fmt.Sprintf("%s: ClientID does not exist", logLabel))
+	}
+
+	rrq := req.GetRequest()
+	lim := req.GetCancelAfterRows()
+
+	if rrq == nil {
+		return nil, stat.Error(codes.InvalidArgument, "request to ReadRows() is missing inner request")
+	}
+
+	t := btc.c.Open(rrq.TableName)
+
+	rowPbs := rrq.Rows
+	var rs bigtable.RowSet
```

Comment:
``` golang
rs := rowSetFromProto(rowPbs)
```

Start with a good default and then override it below to avoid the else.

Reply:
Done.